app/models/info_request.rb

```diff
@@ -14,7 +14,7 @@
 #  awaiting_description                  :boolean          default("false"), not null
 #  prominence                            :string           default("normal"), not null
 #  url_title                             :text             not null
-#  law_used                              :string           default("foi"), not null
+#  law_used                              :string           not null
```

Comment:
This commit ought to explain _why_ the column default "prevents the correct `law_used` from being assigned when the model is initialised." I have some suggestions for later commits, but they _may_ be based on an incorrect understanding of why this was necessary. It's also good to include for future understanding.

Reply:
Will update to include:
> The `#law_used` is assigned using the conditional assignment operator so with this default the value can never be set to the public body legislation key.